### ISSUE TITLE
[Syntax] Examples Clarification

### ISSUE THREAD

**Issue by MarkMoretto** [python-gitlab/python-gitlab#2460]
## Description of the problem, including code/CLI snippet

For some provided examples, initialization of a GitLab object instance appears to use lowercase characters.  No where in the package is a difference in capitalization noted.

Example: [https://python-gitlab.readthedocs.io/en/stable/api-usage-advanced.html#rate-limits](Advanced Usage: Rate Limits) -  

```python
import gitlab
import requests

# This object should be capitalized?
#             |
#             |
#             |
#             V
gl = gitlab.gitlab(url, token, api_version=4)
gl.projects.list(get_all=True, obey_rate_limit=False)
```

Given that Python is a case-sensitive language, `gitlab.Gitlab` != `gitlab.gitlab` unless it's explicitly aliased or noted that either will work.

## Expected Behavior

`N/A`

## Actual Behavior

`N/A`

## Specifications

  - python-gitlab version: `All`
  - API version you are using (v3/v4): `v4`
  - Gitlab server version (or gitlab.com): `gitlab.com`

**Comment by MarkMoretto** [comment]
@nejch Ah, gotcha.  I figured it was just a lowercase thing, but figured I'd ask.

We can close the issue and I'll remind myself to make those minor fixes.

**Comment by JohnVillalovos** [comment]
Rather leave it open until it is fixed. When doing a PR can put `Closes: #2460` in the git commit message and when the PR gets merged it will automatically close this.
